db/schema.rb

```diff
@@ -0,0 +1,26 @@
+ActiveRecord::Schema.define(version: 201_804_041_629_53) do
+  create_table 'users', force: :cascade, options: 'ENGINE=InnoDB DEFAULT CHARSET=utf8' do |t|
```

Comment:
please return schema.rb to initial state. You should not update this file, rails updates it automatically